src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,34 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <>
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
```suggestion
        user && <UserInfo user={post.user} />
```
Consider checking if a user exists

Reply:
Good idea! Done!